src/Engine/ProtoCore/ProtoCore.csproj

```diff
@@ -1,264 +1,36 @@
-    <Compile Include="Lang\Type.cs" />
-    <Compile Include="Lang\CallSite.cs" />
-    <Compile Include="Lang\FunctionEndPoint.cs" />
-    <Compile Include="Lang\FunctionGroup.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="Properties\Resources.Designer.cs">
-      <AutoGen>True</AutoGen>
-      <DesignTime>True</DesignTime>
-      <DependentUpon>Resources.resx</DependentUpon>
-    </Compile>
-    <Compile Include="StringTable.cs" />
-    <Compile Include="Utils\ArrayUtils.cs" />
-    <Compile Include="Utils\ClassUtils.cs" />
-    <Compile Include="Utils\CoreUtils.cs" />
-    <Compile Include="Utils\FileUtils.cs" />
-    <Compile Include="Utils\MathUtils.cs" />
-    <Compile Include="SymbolTable.cs" />
-    <Compile Include="Utils\NodeUtils.cs" />
-    <Compile Include="Utils\StringUtils.cs" />
-    <Compile Include="Utils\Validity.cs" />
+    <ProjectReference Include="..\..\Libraries\DesignScriptBuiltin\DesignScriptBuiltin.csproj" />
+    <ProjectReference Include="..\..\NodeServices\DynamoServices.csproj" />
   </ItemGroup>
   <ItemGroup>
     <Reference Include="Microsoft.CSharp" />
-    <Reference Include="System" />
-    <Reference Include="System.Data" />
     <Reference Include="System.Runtime.Serialization.Formatters.Soap" />
-    <Reference Include="System.Xml" />
-    <Reference Include="WindowsBase" />
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="ExtendedLibraries\BuiltIn.ds" />
-    <None Include="ExtendedLibraries\FunctionObject.ds" />
-    <None Include="Parser\atg\Associative.atg" />
-    <None Include="Parser\atg\End.atg" />
-    <None Include="Parser\atg\Imperative.atg" />
-    <None Include="Parser\atg\Start.atg" />
-    <None Include="Parser\GenerateParser.bat" />
-    <None Include="Parser\Parser.frame" />
-    <None Include="Parser\Scanner.frame" />
-  </ItemGroup>
-  <ItemGroup>
-    <EmbeddedResource Include="Properties\Resources.resx">
-      <Generator>PublicResXFileCodeGenerator</Generator>
-      <LastGenOutput>Resources.Designer.cs</LastGenOutput>
-      <SubType>Designer</SubType>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Properties\Resources.en-US.resx">
-      <Generator>PublicResXFileCodeGenerator</Generator>
-      <LastGenOutput>Resources.en-US.Designer.cs</LastGenOutput>
-      <SubType>Designer</SubType>
-    </EmbeddedResource>
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\Libraries\DesignScriptBuiltin\DesignScriptBuiltin.csproj">
-      <Project>{c0d6dee5-5532-4345-9c66-4c00d7fdb8be}</Project>
-      <Name>DesignScriptBuiltin</Name>
-      <Private>True</Private>
-    </ProjectReference>
-    <ProjectReference Include="..\..\NodeServices\DynamoServices.csproj">
-      <Project>{ef879a10-041d-4c68-83e7-3192685f1bae}</Project>
-      <Name>DynamoServices</Name>
-      <Private>False</Private>
-    </ProjectReference>
+    <Compile Remove="LexUtils.cs" />
+    <Compile Remove="Parser/AssociativeGraph.cs" />
```

Comment:
What does this do? Where is lexutils.cs coming from?

Reply:
These were not compiled in the old format. I decided to keep them around...and not permanently delete them for now.
Not sure if they are useful to keep around though...